lib/js/components/Divider/Divider.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+import Divider from './Divider'; import { PROMINENCES } from './Divider.consts';
+
+<Meta title='components/Divider' component={Divider} />
+
+# Divider
+
+<Story name='interactive'
+	   parameters={{
+		   design: {
+			   type: 'figma',
+			   url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=1552%3A34960',
+		   }
+	   }}
+	   args={{
+		   isVertical: false,
+		   prominence: PROMINENCES.DEFAULT,
+	   }}
+	   argTypes={{
+		   prominence: {
+			   control: { type: 'select', options: Object.values(PROMINENCES) },
+			   defaultValue: PROMINENCES.DEFAULT,
+		   },
+	   }}>
+	{(args) => {
+		return {
+			components: { Divider },
+			props: Object.keys(args),
+			template: '<div style="height: 200px; width: 200px;"><divider :is-vertical="isVertical" :prominence="prominence"/></div>',
```

Comment:
@damianmef @kvas-damian do we have any agreements on how to present components in a pretty way? Current solution works but looks bad.

Also, do we need a separate story for each variant? I see we do that for some of the components but don't know if it's a rule.

Reply:
@rogatty Yes, we have similar situation for contrast background (it's not yet in master):
https://github.com/bethinkpl/design-system/blob/23702055d956516c67a3f2c25a3577cecb0750fa/lib/styles/storybook.scss
just add class in story and css in `storybook.scss`

I don't know findings about separated stories, but it looks clear if we have many stories. You can click from menu every variant and component code is more clear. Additionally you see values of properties and you can send url for specific variant.

Reply:
> Also, do we need a separate story for each variant? I see we do that for some of the components but don't know if it's a rule.
Variants are usually requested by UX, I don't see this requirement for Divider: https://miro.com/app/board/uXjVO_rTPmE=/?moveToWidget=3458764522565713965&cot=10 but you can confirm that